src/vs/base/parts/quickinput/browser/media/quickInput.css

```diff
@@ -6,7 +6,7 @@
 .quick-input-widget {
 	position: absolute;
 	width: 600px;
-	z-index: 2000;
+	z-index: 4000;
```

Comment:
fyi @sbatten different approach to fixing https://github.com/microsoft/vscode/issues/115267 because it is now expected that quick pick can overlay the title bar (when the title menu shows)